HogeFuga/app/src/main/java/example/watarusuzuki/android_example_using_danger/MainActivity.java

```diff
@@ -0,0 +1,37 @@
+package example.watarusuzuki.android_example_using_danger;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
+
+    private Intent mIntent;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        while (true) {
+            Intent intent = getIntent();
+            intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+            startActivity(intent);
+            if (intent.hasExtra("hoge")) {
+                convertDaysToMilliseconds(1);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/d44a3ff3897f5daace1e0ce43cb9b80e4928e0e5/HogeFuga/app/src/main/java/example/watarusuzuki/android_example_using_danger/MainActivity.java#L22"></span>Return value of convertDaysToMilliseconds(int) ignored, but method has no side effect</td>
    </tr>
  </tbody>
</table>